src/main/scala/diplomaticobjectmodel/model/OMISA.scala

```diff
@@ -82,10 +82,17 @@ object OMISA {
     }
 
     val addressTranslationModes = xLen match {
```

Comment:
This is just stylistic, but you could do `(xLen, pgLevels) match {` to flatten this structure and only have one error message ("invalid XLEN/PgLevels combination")